packages/logger/src/logger-otel-transport.ts

```diff
@@ -29,13 +29,6 @@ export const attachLoggerOtelTransport = (
       ...attributes,
     }).reduce(
       (acc, [key, value]) => {
-        /**
-         * Prune empty keys, to save bandwidth
-         */
-        if (!value) {
-          return acc;
-        }
-
```

Comment:
Maybe we should just check if the value is undefined, so that we don't send empty objects?

```js
if (value !== undefined) {
  return acc;
}
```

Reply:
but won't it cause the same problem? I mean if I have nested field e.g `channelSlug: payload.order?.channelSlug` is `undefined` OTEL logs won't send `channelSlug` and in our log dashboard we will have `noData` view. Maybe we can fallback empty values to `null` instead - but this requires remembering about that every time we log something 🤔